lib/Parse/ParseDecl.cpp

```diff
@@ -7100,6 +7104,10 @@ void Parser::parseTopLevelAccessors(
     );
   }
 
+  if (Tok.is(tok::r_brace)) {
```

Comment:
Probably it doesn't mean anything at this point, but this should check if the first token was `l_brace`.
```cpp
bool hadLBrace = consumeIf(tok::l_brace);
...
if (hadBrace && Tok.is(tok::r_brace))

Reply:
I figured this is generated by us anyway, but can change if you want 😅